meta/model/src/main/kotlin/types.kt

```diff
@@ -1,7 +1,61 @@
 package dev.inkremental.meta.model
 
+import com.squareup.kotlinpoet.*
+import kotlinx.serialization.Serializable
+import java.io.Serializable as JvmSerializable
+
 // Inner Any can be String and Boolean for __viewAlias and Boolean for others
 typealias InkrementalQuirks = MutableMap<String, Map<String, Any?>>
+typealias InkrementalTransformers = Map<String, Map<String, List<DslTransformer>>>
 
 enum class InkrementalType { SDK, LIBRARY }
 enum class InkrementalPlatform { ANDROID, IOS }
+
+@Serializable
+sealed class DslTransformer : JvmSerializable {
+    /**
+     * @return True if you want this transformer to stop execution of regular dsl generator
+     */
+    open fun handleTransformersForDsl(builder: FunSpec.Builder, attrModel: AttrModel, attr: MemberName): Boolean {
+        return false
+    }
+    /**
+     * @return True if you want this transformer to stop execution of regular dsl generator
+     */
+    open fun handleTransformersForAttrSetter(builder: CodeBlock.Builder, owner: ViewModel, v: String, setterName: String, argAsParam: String): Boolean {
+        return false
+    }
+
+    @Serializable
+    object FloatPixelToDipSizeTransformer : DslTransformer() {
+
+        override fun handleTransformersForDsl(builder: FunSpec.Builder, attrModel: AttrModel, attr: MemberName): Boolean {
+            if (attrModel.type.argType.toString() == "kotlin.Float"){
```

Comment:
Isn't there constants for primitive `ClassName`s in kotlinpoet, so that we can compare `argType` to it, not it's string representation?

Reply:
Sorry for not searching for it aforehead, I've meant
`val com.squareup.kotlinpoet.FLOAT: ClassName`
https://github.com/square/kotlinpoet/blob/01497dd2369b4a1e71cad0fd47c9ce4dec0f0b82/kotlinpoet/src/main/java/com/squareup/kotlinpoet/TypeName.kt#L230